axelor-sale/src/main/resources/views/SaleOrder.xml

```diff
@@ -321,13 +321,13 @@
         </field>
         <field name="$invoicingState" showTitle="false" readonly="true"
           showIf="statusSelect >= 3" css="label-bold bold" colSpan="3">
```

Comment:
add attributes for this field
type="integer" 
if="__config__.app.isApp('supplychain')" 
if-module="axelor-supplychain"